scripts/linux/testHelper.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+function clean_up() {
```

Comment:
Is this file new? Don't existing tests need this?

Reply:
yes.  this file is new and it is extracted from runE2ETests.sh.  Currently it is used only by connectivityTest.sh.  In the future, runE2ETests.sh can use it as well.